src/transformers/models/mobilenet_v1/configuration_mobilenet_v1.py

```diff
@@ -0,0 +1,129 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" MobileNetV1 model configuration"""
+
+from collections import OrderedDict
+from typing import Mapping
+
+from packaging import version
+
+from ...configuration_utils import PretrainedConfig
+from ...onnx import OnnxConfig
+from ...utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+MOBILENET_V1_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "google/mobilenet_v1_1.0_224": "https://huggingface.co/Matthijs/mobilenet_v1_1.0_224/resolve/main/config.json",
+    "google/mobilenet_v1_0.75_192": "https://huggingface.co/Matthijs/mobilenet_v1_0.75_192/resolve/main/config.json",
```

Comment:
```suggestion
    "google/mobilenet_v1_1.0_224": "https://huggingface.co/google/mobilenet_v1_1.0_224/resolve/main/config.json",
    "google/mobilenet_v1_0.75_192": "https://huggingface.co/google/mobilenet_v1_0.75_192/resolve/main/config.json",
```

Reply:
Whoops, thanks for catching that. I noticed that I made the same mistake in the MobileNetV2 config, so I've included that in this PR too (it's only a small change).